src/makeCalculator.js

```diff
@@ -4,7 +4,28 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  let result = 0;
+
+  return {
+    get result() {
+      return result;
+    },
+
+    add: (number) => result + number,
+    subtract: (number) => result - number,
+    multiply: (number) => result * number,
+    divide: (number) => result / number,
+    operate: function (callback, number) {
```

Comment:
You can use the method shorthand syntax for defining object methods. According to the checklist [CODE KNOWLEDGE], you don't need to use the `function` keyword here.